src/prompt/unfinished_run_state.go

```diff
@@ -0,0 +1,58 @@
+package prompt
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Originate/exit"
+	humanize "github.com/dustin/go-humanize"
+	survey "gopkg.in/AlecAivazis/survey.v1"
+)
+
+var (
+	// ResponseTypeAbort stands for the user choosing to abort the unfinished run state
+	ResponseTypeAbort = "abort"
+	// ResponseTypeContinue stands for the user choosing to continue the unfinished run state
+	ResponseTypeContinue = "continue"
+	// ResponseTypeDiscard stands for the user choosing to discard the unfinished run state
+	ResponseTypeDiscard = "discard"
+	// ResponseTypeQuit stands for the user choosing to quit the program
+	ResponseTypeQuit = "quit"
+	// ResponseTypeSkip stands for the user choosing to continue the unfinished run state by skipping the current branch
+	ResponseTypeSkip = "skip"
+)
+
+// AskHowToHandleUnfinishedRunState prompts the user for how to handle the unfinished run state
+func AskHowToHandleUnfinishedRunState(command, endBranch string, endTime time.Time, canSkip bool) string {
+	formattedOptions := map[string]string{
+		ResponseTypeAbort:    fmt.Sprintf("Abort the `%s` command", command),
+		ResponseTypeContinue: fmt.Sprintf("Continue the `%s` command (conficts are resolved)", command),
+		ResponseTypeDiscard:  "Discard the unfinished state and run the new command",
+		ResponseTypeQuit:     "Quit without running anything.",
```

Comment:
I'd remove the period at the end of the string here, the other entries don't have one either.